MAUI/DataGrid/Getting-started.md

```diff
@@ -0,0 +1,341 @@
+
+![.Net MAUI DataGrid](Images\getting-started\basic-datagrid.jpg)
+
+## Defining columns
+
+By default, the SfDataGrid automatically creates columns for all the properties in the data source. The type of the column generated depends on the type of data in the column. When the columns are auto-generated, handle the `SfDataGrid.AutoGeneratingColumnMode` mode to customize or cancel the columns before they are added to the columns collection in the SfDataGrid.
+ 
+The columns can be manually defined by setting the`SfDataGrid.AutoGenerateColumnsMode` property to None and by adding the `GridColumn` objects to the `SfDataGrid.Columns` collection. It can be done from both XAML and code. The following code example illustrates this:
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+            ColumnSizer="Star"
+            AutoGenerateColumnsMode="None"
+            ItemsSource="{Binding OrderInfoCollection}">
+
+    <syncfusion:SfDataGrid.Columns >
+        <syncfusion:DataGridTextColumn HeaderText="ID" 
+                                   MappingName="OrderID" />
+        <syncfusion:DataGridTextColumn HeaderText="Customer "
+                                   MappingName="CustomerID" />
+        <syncfusion:DataGridTextColumn MappingName="Customer" />
+        <syncfusion:DataGridTextColumn HeaderText="Country"
+                                   MappingName="ShipCountry" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.AutoGenerateColumnsMode = AutoGenerateColumnsMode.None;
+
+DataGridTextColumn orderIdColumn = new DataGridTextColumn ();
+orderIdColumn.MappingName = "OrderID";
+orderIdColumn.HeaderText = "Order ID";
+
+DataGridTextColumn customerIdColumn = new DataGridTextColumn ();
+customerIdColumn.MappingName = "CustomerID";
+customerIdColumn.HeaderText = "Customer ID";
+
+DataGridTextColumn customerColumn = new DataGridTextColumn ();
+customerColumn.MappingName = "Customer";
+customerColumn.HeaderText = "Customer";
+
+DataGridTextColumn countryColumn = new DataGridTextColumn();
+countryColumn.MappingName = "ShipCountry";
+countryColumn.HeaderText = "Ship Country";
+
+dataGrid.Columns.Add (orderIdColumn);
+dataGrid.Columns.Add (customerIdColumn);
+dataGrid.Columns.Add (customerColumn);
+dataGrid.Columns.Add (countryColumn); 
+{% endhighlight %}
+{% endtabs %}
+
+## Sorting
+
+In SfDataGrid sorting can be done on its data by setting the `SfDataGrid.SortingMode` property to single,multiple or none.
+ 
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid SortingMode="Single" />
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.SortingMode=DataGridSortingMode.Single; 
+{% endhighlight %}
+{% endtabs %}
+
+Run the application and touch the header cell to sort the data and the following output will be displayed.
+ 
+
+
+Sorting can also be configured by adding the column to the `SfDataGrid.SortColumnDescription` collection as follows.
```

Comment:
Sorting can also be configured by adding the column to the `SfDataGrid.SortColumnDescription` collection as follows: